V3 (#117)
balena-io/landr: README.md

<h1 align="center">
    Landr
</h1>

> 

Reviewer: A short explanation fo what landr does would be ideal here

Author: The Readme needs some love for sure. For now, I added the previous subtitle "Build a website for your software projects with one command.".

## 🚀 Quick start

1. Navigate to the repository

> 2. run `GITHUB_TOKEN=[your token] npx landr deploy`

Reviewer: Can you clarify what permissions the token needs?

Author: Great suggestion. Also opened this one https://github.com/balena-io-modules/scrutinizer/issues/56 👍

3. Landr will push the changes to the `gh-pages` branch.

> We encourage using `npx` as opposed to installing landr globally. `npx` comes bundled with NPM version 5.2+. More info can be found [here](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b).

## Alternatively using Landr as a dependency

1. Install landr

`npm install landr`

> 1. Add the a npm script entry in your `package.json` file

Reviewer: ```suggestion
1. Add the following to the script entry in the project's `package.json` file
```

```json
  "scripts": {
    "landr": "landr deploy",
    "landr-preview": "landr preview"
  }
```

3. Run the CLI
`GITHUB_TOKEN=[your token] npm run landr`

> 1. Landr will push the changes to the `gh-pages` branch.

Reviewer: The phrase `the changes` is quite ambiguous, you could probable change this to 
```suggestion
1. Landr will push the generated files to the `gh-pages` branch.
```
